libmscore/rendermidi.cpp

```diff
@@ -731,6 +735,478 @@ bool Score::isSubdivided(ChordRest* chord, int swingUnit)
             return false;
       }
 
+void renderTremolo(Chord *chord, QList<NoteEventList> & ell){
```

Comment:
if you commit your indentation changes, I can merge that into my own to avoid us having conflicting indentation changes.


> On 18 May 2015, at 14:20, Nicolas Froment <notifications@github.com> wrote:
> 
> 
> 
> //---------------------------------------------------------
>  We use if (nospace) { and idem for while, case etc... I started to reindent it so don't bother for this one.
> 
> —
>